types/react-redux/index.d.ts

```diff
@@ -164,7 +164,7 @@ interface MergeProps<TStateProps, TDispatchProps, TOwnProps, TMergedProps> {
     (stateProps: TStateProps, dispatchProps: TDispatchProps, ownProps: TOwnProps): TMergedProps;
 }
 
-interface Options extends ConnectOptions {
+interface Options<TStateProps = any, TOwnProps = any, TMergedProps = any> extends ConnectOptions {
```

Comment:
I think it may depend on your compiler options -- but the build was failing (I don't think I can link to it?) due to the arguments not being provided.

I'll change the default to {}